lib/livebook/session.ex

```diff
@@ -62,7 +62,8 @@ defmodule Livebook.Session do
           notebook_name: String.t(),
           file: FileSystem.File.t() | nil,
           images_dir: FileSystem.File.t(),
-          created_at: DateTime.t()
+          created_at: DateTime.t(),
+          memory_usage: map() | nil
```

Comment:
We can document the exact map type in `Livebook.Runtime` too!

```suggestion
          memory_usage: Livebook.Runtime.memory_usage() | nil
```